backend/emails/shipping_confirmation.html

```diff
@@ -0,0 +1,205 @@
+                    margin-bottom: 20px;
+                    font-weight: 300 !important;
+                }
+                .raster {
+                    width: 40px !important;
+                    height: 40px !important;
+                    margin: 10px 0!important;
+                }
+                #logo {
+                    margin: 0 auto !important;
+                    width: 252px !important;
+                }
+                #footer p, #footer a {
+                    line-height: 20px !important;
+                    font-size: 15px !important;
+                }
+            }
+        </style>
+    </head>
+    <body style="max-width:580px;margin:0 auto;padding:20px;font-family:'Helvetica Neue', HelveticaNeue, Helvetica, 'Lucida Grande', arial, sans-serif;font-weight:300;font-size:14px;color:#222222;line-height:20px;">
+        
+        <div style="width:auto;margin:0 0 30px 0;" id="logo">
+            <a href="https://elementary.io" title="elementary">
+                <img src="https://elementary.io/images/logotype.png" height="28" alt="elementary" title="elementary">
+            </a>
+        </div>
+
+        <div style="width:auto;">
+            
+<h2 style="font-weight:300;font-size:30px;line-height:40px;">Your Order #PF1098494 has been sent out!</h2>
+
+<p style="font-weight:300;font-size:15px;color:#222222;line-height:20px;">
+    <strong>Hey there,</strong>
+</p>
+
+<p style="font-weight:300;font-size:15px;color:#222222;line-height:20px;">
+    Just letting you know that your order #PF1098494 has been sent out!
+</p>
+
+<p style="font-weight:300;font-size:15px;color:#222222;line-height:20px;margin-bottom:5px;">
+    <strong>Here's tracking number:</strong>
+</p>
+
+<p style="background-color:#f8f8f8;margin-top:0;margin-bottom:0;padding:20px;font-weight:100;font-size:30px;color:#222222;line-height:30px;">
+    9400110897700072490282<br>
+    <a href="https://tools.usps.com/go/TrackConfirmAction!input.action?tLabels=9400110897700072490282" style="outline:0;font-weight:300;font-size:18px;color: #08c;line-height:30px;text-decoration:none;">(USPS First Class Mail)</a>
+</p>
+
+<p style="margin-top:10px;font-weight:300;font-style:italic;font-size:15px;color:#222222;line-height:20px;">
+    Note: Tracking information can take up to 48 hours to be updated after the order is shipped.
+</p>
+
+
+<p style="font-weight:300;font-size:15px;color:#222222;line-height:20px;">
+    Estimated delivery Aug 22nd, 2016 to Aug 24th, 2016</p>
+
+<p style="font-weight:300;font-size:15px;color:#222222;line-height:20px;">
+    Here's what we shipped in this order:
+</p>
+
+<table width="100%" cellspacing="0" cellpadding="0">
+    <tbody>
+        <tr>
+            <td width="25%" valign="top" style="padding-top:10px;padding-bottom:10px;">
+                <img src="https://d1yg28hrivmbqm.cloudfront.net/files/6aa/6aa7c59b099906c64a9983a3b5377080_thumb.png" width="100%" alt="" style="background-color:#f0f0f0;">
```

Comment:
Yeah I'm not sure what's the best way to get this but it should be the same image as the thumbnail in the store.